libs/ledgerjs/packages/hw-app-polkadot/src/Polkadot.ts

```diff
@@ -176,4 +190,10 @@ export default class Polkadot {
       };
     });
   }
+
+  async getMajorAppVersion(): Promise<number> {
```

Comment:
It can be declared as `private` as it is not used outside of the class.

Reply:
fixed